src/lib/ibcclient.ts

```diff
@@ -257,6 +272,27 @@ export class IbcClient {
     this.logger = logger ?? new NoopLogger();
   }
 
+  public revisionHeight(height: number): Height {
+    return Height.fromPartial({
+      revisionHeight: new Long(height),
+      revisionNumber: this.revisionNumber,
+    });
+  }
+
+  public ensureRevisionHeight(height: number | Height): Height {
```

Comment:
What if it's a `Height` but the `revisionNumber`s don't match?

Reply:
I guess I can assert that as well. Good point